bay/images/Dockerfile.ci-builder

```
@@ -1,40 +1,64 @@
 FROM hashicorp/terraform:light AS terraform
-FROM integratedexperts/ci-builder:latest
+FROM uselagoon/php-7.4-cli:latest
```

You could probably just replace this with 

```suggestion
FROM php:7.4-cli
```

---

This would ditch all the goofy ssh key shenanigans

---

👍  I'll give that a go.